e2e/tests/popupEscHandling-example.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react'
+import { Button, Dropdown, Popup } from '@stardust-ui/react'
+
+const inputItems = ['Bruce Wayne', 'Natasha Romanoff', 'Steven Strange']
+
+export const selectors = {
+  popupTriggerId: 'trigger',
+  popupContentClass: Popup.slotClassNames.content,
+  dropdownTriggerClass: Dropdown.slotClassNames.triggerButton,
+}
+
+const PopupExample = () => (
+  <Popup
+    trigger={<Button id={selectors.popupTriggerId} content="Open popup" style={{ margin: 50 }} />}
+    content={{
+      content: (
+        <React.Fragment>
+          <div>Hello from inner popup!</div>
+
+          <Dropdown
+            items={inputItems}
+            placeholder="Select your hero"
+            getA11ySelectionMessage={{
+              onAdd: item => `${item} has been selected.`,
+            }}
+          />
+        </React.Fragment>
+      ),
+    }}
+  />
+)
+
+export default PopupExample
```

Comment:
Should name somehow match filename?

Reply:
actually, think that it will be way to easier if we will name example component types as just `Example` - as, essentially, there are no scenarios where specific name will be helpful, while just `Example` as name will save headache from name refactoring